forward/fwd_websocket_test.go

```diff
@@ -150,6 +149,52 @@ func (s *FwdSuite) TestWebsocketRequestWithOrigin(c *C) {
 	c.Assert(resp, Equals, "ok")
 }
 
+func (s *FwdSuite) TestWebsocketRequestWithQueryParams(c *C) {
+	f, err := New()
+	c.Assert(err, IsNil)
+
+	upgrader := gorillawebsocket.Upgrader{}
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		conn, err := upgrader.Upgrade(w, r, nil)
+		if err != nil {
+			return
+		}
+		defer conn.Close()
+		c.Assert(r.URL.Query().Get("query"), Equals, "test")
+		for {
```

Comment:
Are we busy-waiting here consuming lots of CPU possibly?

Is there a risk we never finish the for loop in case of a bug and have to wait until a timeout hits?

I'm lacking web-socket knowledge, so basically just uttering vague concerns.